bciers/apps/reporting/src/app/components/finalReview/reviewDataFactory/productionDataFactoryItem.ts

```diff
@@ -8,10 +9,19 @@ const productionDataFactoryItem: ReviewDataFactoryItem = async (
 ) => {
   const productionData = await getProductionData(versionId, facilityId);
 
+  const facilityType = (await getFacilityReportDetails(versionId, facilityId))
+    .facility_type;
+  const productionMethodology = ["Small Aggregate", "Medium Facility"].includes(
+    facilityType,
+  )
+    ? ["Not Applicable", "OBPS Calculator", "other"]
+    : ["OBPS Calculator", "other"];
```

Comment:
This logic should be in the `buildProductionDataSchema` function, which has the responsiblity of building the schema.
Instead of passing it the `productionMethodology` array, we can pass the `facilityType` to it directly